src/api/data/tiltak.ts

```diff
@@ -56,8 +56,16 @@ export const koordinatorListSchema = z.array(koordinatorSchema)
 
 export const gjennomforingerSchema = z.array(gjennomforingSchema)
 
+export const tiltakErrorSchema = z.object({
+	status: z.number(),
+	title: z.string(),
+	detail: z.string().nullable()
+})
+
 export type Gjennomforing = z.infer<typeof gjennomforingSchema>
 
+export type TiltakError = z.infer<typeof tiltakErrorSchema>
```

Comment:
Er denne bare til mocking? isåfall så ville jeg laget en vanlig typescript interface i mock mappa eller potensielt ikke typa det opp i det hele tatt. Zod er mer for å få sikker schemavalidering av responsen som kommer fra api.

Reply:
eiei, dette var for å få testet før vi snakket sammen, skal fjerne!